Assets/Scripts/Gameplay/GameplayView.cs

```diff
@@ -1,13 +1,11 @@
 using Agate.MVC.Base;
-//using TankU.Module.SFX;
-using System.Collections;
-using System.Collections.Generic;
+using TankU.Module.Timer;
 using UnityEngine;
 
 namespace TankU.Gameplay
 {
     public class GameplayView : BaseSceneView
     {
-        //[SerializeField] public SFXView SFXView;
```

Comment:
Overall OK for Push. SFXView is for me later, I should've used TODO here, but yeah I'll add that later on.